03 Tests samples/src/components/NameForm/NameForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+
+interface MyState {
+  value: string;
+  clicked: boolean;
+}
+export class NameForm extends React.Component<{}, MyState> {
+	constructor(props: MyState) {
+		super(props);
+		this.state = { value: '' , clicked: false };
+
+		this.handleChange = this.handleChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	handleChange(event) {
+		this.setState({ value: event.target.value });
+	}
+
+	handleSubmit(event) {
```

Comment:
If you convert this to lambda function then the line 14 can be deleted, and it is less code! Try it!

Reply:
I don't really know what do you mean.